### **Figure 1**

To generate the SNPs for the S region for the variants of concern (VOCs), the `manuscript_SpikeID` conda environment must be loaded as it includes the tool, [`snipit`](https://github.com/aineniamh/snipit). Alternatively, the bash script checks if it is installed, installing it using `pip` if it isn't. The file `nextclade_vocs.aligned.S.renamed.fasta` must exist in the `FASTAs` directory. This FASTA file was generated by selecting a representative nucleotide sequence from each variant, aligning [`Nexclade`](https://clades.nextstrain.org/), and masking the regions outside of the Spike S1 domain to preserve the genome coordinates. The masked alignment is used as input for running the [`run-snipit.sh`](https://github.com/BakelLab/manuscript_SpikeID/blob/main/figures/figure1/scripts/run-snipit.sh) script, which is found in `figures/figure1/scripts`, generates the corresponding PDF and CSV files, storing them in `figures/figure1/figures` as 

**<u>Output:</u>**

```bash
figures/figure1/figures/figure1.pdf
figures/figure1/figures/figure1_snps.csv
```

